TicketingContract.sol

```diff
@@ -2,74 +2,59 @@
-// "0x474d542b33000000000000000000000001075b2282f05255534b534101050008", 
+// "0x474d542b33000000000000000000000001075b2282f05255534b534101050008",
 // "0x474d542b33000000000000000000000001075b2282f05255534b534101050009",
 // "0x474d542b33000000000000000000000001075b2282f05255534b53410105000a",
 // "0x474d542b33000000000000000000000001075b2282f05255534b53410105000b",
 // "0x474d542b33000000000000000000000001075b2282f05255534b53410105000c",
 // "0x474d542b33000000000000000000000001075b2282f05255534b53410105000d",
-// "0x474d542b33000000000000000000000001075b2282f05255534b53410105000e", 
-// "0x474d542b33000000000000000000000001075b2282f05255534b53410105000f", 
-// "0x474d542b33000000000000000000000001075b2282f05255534b534101050010", 
-// "0x474d542b33000000000000000000000001075b2282f05255534b534101050011", 
-// "0x474d542b33000000000000000000000001075b2282f05255534b534101050012", 
-// "0x474d542b33000000000000000000000001075b2282f05255534b534101050013", 
-// "0x474d542b33000000000000000000000001075b2282f05255534b534101050014"], 
-// "FIFA WC2018", 
-// "SHANKAI", 
-// "0x0D590124d2fAaBbbdFa5561ccBf778914a50BCca", 
+// "0x474d542b33000000000000000000000001075b2282f05255534b53410105000e",
+// "0x474d542b33000000000000000000000001075b2282f05255534b53410105000f",
+// "0x474d542b33000000000000000000000001075b2282f05255534b534101050010",
+// "0x474d542b33000000000000000000000001075b2282f05255534b534101050011",
+// "0x474d542b33000000000000000000000001075b2282f05255534b534101050012",
+// "0x474d542b33000000000000000000000001075b2282f05255534b534101050013",
+// "0x474d542b33000000000000000000000001075b2282f05255534b534101050014"],
+// "FIFA WC2018",
+// "SHANKAI",
+// "0x0D590124d2fAaBbbdFa5561ccBf778914a50BCca",
 // "0xFE6d4bC2De2D0b0E6FE47f08A28Ed52F9d052A02",
 // "0x2e558Bc42E2e37aB638daebA5CD1062e5b9923De"
 
 pragma solidity ^0.4.17;
 contract TicketPro
 {
-    mapping(address => bytes32[]) inventory;
-    uint16 ticketIndex = 0; //to track mapping in tickets
+    mapping(address => uint256[]) inventory;
     address organiser;
     address paymaster;
-    uint numOfTransfers = 0;
     string public name;
     string public symbol;
     uint8 public constant decimals = 0; //no decimals as tickets cannot be split
 
-    event Transfer(address indexed _to, uint16[] _indices);
-    event TransferFrom(address indexed _from, address indexed _to, uint16[] _indices);
-    event Trade(address indexed seller, uint16[] ticketIndices, uint8 v, bytes32 r, bytes32 s);
-    event PassTo(uint16[] ticketIndices, uint8 v, bytes32 r, bytes32 s, address indexed recipient);
+    event Transfer(address indexed _to, uint256[] tickets);
+    event TransferFrom(address indexed _from, address indexed _to, uint256[] tickets);
+    event Trade(address indexed seller, uint256[] tickets, uint8 v, bytes32 r, bytes32 s);
+    event PassTo(uint256[] tickets, uint8 v, bytes32 r, bytes32 s, address indexed recipient);
```

Comment:
Need to remove arrays from events. They don't convert into valid values. Instead use counts or the first ID of a ticket array for disambiguation.